core/src/main/java/com/linecorp/armeria/common/thrift/ThriftCallbackCompletableFuture.java

```diff
@@ -0,0 +1,22 @@
+package com.linecorp.armeria.common.thrift;
```

Comment:
I think I need to move these callbacks to better place (but no idea). WDYT @trustin ?